src/ecalc/libraries/libecalc/common/libecalc/core/graph_result.py

```diff
@@ -343,3 +388,43 @@ def get_results(self) -> EnergyCalculatorResult:
             emission_results=self.emission_results,
             variables_map=self.variables_map,
         )
+
+    def get_pressures_from_temporal_models(
```

Comment:
if we want to treat all special properties equally wrt formatting in output/results, might be better to have a method for that, where we just send in an "internal/core property" that is formatted to standard representation, instead of adding the logic to find the correct property here ...

Reply:
Where do you suggest to put the method, core/result/base?

Reply:
Could it be separated into getting the temporal expression and evaluating it? We have a TemporalExpression class in `libecalc/common/temporal_model.py` that can evaluate temporal expressions.

Getting the temporal expression could then be moved to the component dto

Reply:
I agree. Would be nice to add that now then imo.